TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/OutTake.kt

```diff
@@ -0,0 +1,30 @@
+package org.firstinspires.ftc.teamcode.api
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode
+import com.qualcomm.robotcore.hardware.Servo
+
+object OutTake : API() {
+    // servo
+    lateinit var outTakeBelt: Servo
+        private set
+
+    // how much the conveyor belt will move to drop one pixel
+    private const val outTakePosition: Double = 8.0
+
+
+    override fun init(opMode: OpMode) {
+        super.init(opMode)
+
+        this.outTakeBelt = hardwareMap.get(Servo::class.java, "ServoBelt")
```

Comment:
Could you rename "ServoBelt" to something that better describes the outtake servo? It being a servo is implied in the configuration.

Reply:
The intake branch uses "inTakeConvey," so perhaps "outTakeConvey?" I'd like your input on this.

Reply:
done and I like the change the better naming convention was created second so it makes sense to update to outTakeConvey